pkg/host-disk/host-disk.go

```diff
@@ -88,13 +89,14 @@ func dirBytesAvailable(path string) (uint64, error) {
 	return stat.Bavail * uint64(stat.Bsize), nil
 }
 
-func createSparseRaw(fullPath string, size int64) error {
+// Use named return err to allow checking Close() errors in defer
```

Comment:
This is new? I do not remember seeing this before.
Comments tend to rot. Unless used to explain something non-obvious from the code itself I prefer to avoid them. I think it should be obvious that if a named return value is given it is used for some purpose. It may be because of the close or other reasons.
We surely do not need to own or limit this to the defer close handling.

If you do not mind, I prefer these out.

Reply:
This comment is important so I prefer to leave it. If someone if the future decides to change the named variable to "regular" returned variable , (e.g., because "it's not pretty "), nothing obvious will break, it will just not work, and it will be very hard to track it down.

I will squash some of the similar commits.

Reply:
You assume that they will look or trust the comment. Comments do not protect, type checking, reviews, linters and unit tests protect.

But I leave it to you and other reviewers to choose.

Reply:
Will remove from all.